app/views/site_informations/about.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Title</title>
+</head>
+<body class = "backroundcolorAbout">
```

Comment:
Codeverdopplung, da bereits durch `app/views/layouts/application.html.erb` gegeben. In den Views brauchst du kein `html.head`, `body`, etc. einfügen.

https://edgeguides.rubyonrails.org/layouts_and_rendering.html#understanding-yield